README.md

```diff
@@ -1,6 +1,22 @@
 # keycloak-theme
 VSHN Keycloak IDP Theme
 
-## Local development
+## Local Development
 
-Run `docker compose up` to start a local keycloak with the vshn theme.
+Run `docker-compose up` to start a local keycloak with the vshn theme.
+
+## Deploy to Your Instance
+
+1. Merge your Pull Request with the theme to master
+1. Tag it with
+    ```bash
+    git tag -am v1.x.y
+    git push --tags
+    ```
+1. This will trigger a deployment on https://quay.io/repository/vshn/keycloak-theme?tab=tags
+1. Add the tag of the version to your extraInitContainers ([see codecentric helm chart documentation](https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#providing-a-custom-theme))
+    ```yaml
+    extraInitContainers: |
+      - name: theme-provider
+        image: quay.io/vshn/keycloak-theme:v1.0.0
+    ```
```

Comment:
```suggestion
1. This will trigger an image push on https://quay.io/repository/vshn/keycloak-theme?tab=tags
1. Add the tag of the version to your extraInitContainers ([see codecentric helm chart documentation](https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#providing-a-custom-theme))
    ```yaml
    extraInitContainers: |
      - name: theme-provider
        image: quay.io/vshn/keycloak-theme:v1.x.x
    ```
```